docs/software/media.md

```diff
@@ -556,13 +556,20 @@ Also Installs:
     - Enter the details of your path (See the image example below to add your USB drive music folder to Ampache).
     - Select the `add catalogue` button.
 
-    Remark: For Ampache to access custom directories, you must ensure read access to the folder, e.g. by:
+    !!! note "Check access rights on local custom directories"
+        For Ampache to access custom directories, you must ensure read access to the folder, e.g. by:
 
-    ```sh
-    chmod -R 775 /my/directory
-    ```
+        ```sh
+        chmod -R 775 /my/directory
+        ```
```

Comment:
Actually this does not grants read access to all users and write access to all group members + execute permissions to all files recursively, which isn't what we aim here. What about `chmod -R a+r /my/directory` to grant all users read access only, without touching any other modes?